tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -434,6 +434,13 @@ decompress_chunk_create_tuple(DecompressChunkState *state)
 
 		ExecClearTuple(slot);
 
+		/* some technical columns may left unpopulated in the slot;
+		 * setting isnull prevents issues for row expressions */
+		for (i = 0; i < slot->tts_tupleDescriptor->natts; i++)
+		{
+			slot->tts_isnull[i] = true;
+		}
```

Comment:
Suggest to use the existing function. It does some more work, but I think it is better to rely on PostgreSQL code for this.
```suggestion
		ExecStoreAllNullTuple(slot);
```